### PR TITLE
Change bookmark button to always visible

### PR DESCRIPTION
**Pull request by Pasithea0**
Change bookmark button to always visible

 - [ x] I have read and agreed to the [code of conduct](https://github.com/sussy-code/smov/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [ x] I have read and complied with the [contributing guidelines](https://github.com/sussy-code/smov/blob/dev/.github/CONTRIBUTING.md).
 - [ x] I have tested all of my changes.

### CODE REVIEW

File: src/components/media/MediaCard.tsx [hunk 1]
```diff
@@ -145,11 +145,11 @@ function MediaCardContent({
             </>
           ) : null}
 
-          {searchQuery.length > 0 ? (
+          {true && (
```
Comment (ztpn) [inline src/components/media/MediaCard.tsx:148]: Redundant conditional rendering?

**Review by ztpn — CHANGES_REQUESTED** [review]

**Comment by itzCozi** [comment]
Lemme get a preview link

**Comment by Pasithea0** [comment]
https://bookmarks-always-visible.envyflix.pages.dev/
Bookmarks will always show for smaller screens, assuming they are on mobile and don't have a mouse to hover. Otherwise PR [44dddea](https://github.com/sussy-code/smov/pull/82/commits/44dddead7cabc2431566012308282b08a19f0d3e) is just always show, even on desktop.

**Comment by ztpn** [comment]
we have an isMobile() somewhere

**Comment by ztpn** [comment]
```typescript
import { useIsMobile } from "@/hooks/useIsMobile";
const { isMobile } = useIsMobile();
```

**Pull request merged**